packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import {
+  Button,
+  css,
+  spacing,
+  Link,
+  Icon,
+} from '@mongodb-js/compass-components';
+
+import type { RootState } from '../../../modules';
+import { runAggregation } from '../../../modules/aggregation';
+import { toggleOptions } from '../../../modules/options';
+
+const containerStyles = css({
+  display: 'grid',
+  gap: spacing[2],
+  gridTemplateAreas: '"run options"',
+});
+
+const runButtonStyles = css({
+  gridArea: 'run',
+});
+
+const optionsButtonStyles = css({
+  gridArea: 'options',
+  backgroundColor: 'transparent',
+  border: 'none',
+  display: 'inline',
+});
+
+const optionStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const PipelineActions: React.FunctionComponent<PipelineActionsProps> = ({
+  isOptionsVisible,
+  onRunAggregation,
+  onToggleOptions,
+}) => {
+  const optionsIcon = isOptionsVisible ? 'CaretDown' : 'CaretRight';
+  return (
+    <div className={containerStyles}>
+      <Button
+        data-testid="pipeline-toolbar-run-button"
+        className={runButtonStyles}
+        variant="primary"
+        size="small"
+        onClick={() => {
+          onRunAggregation();
+        }}
+      >
+        Run
+      </Button>
+      <Link
+        as="button"
+        className={optionsButtonStyles}
+        data-testid="pipeline-toolbar-options-button"
+        hideExternalIcon={true}
+        onClick={() => onToggleOptions()}
+      >
+        <div className={optionStyles}>
+          {isOptionsVisible ? 'Less' : 'More'} Options{' '}
```

Comment:
Because the text size here changes, the Run button above jerks a little when clicking, should we add some min-width to the whole button here so this doesn't happen?